packages/editor-ui/src/components/banners/BannerStack.vue

```diff
@@ -1,38 +1,59 @@
-<script setup lang="ts">
+<script lang="ts">
 import NonProductionLicenseBanner from '@/components/banners/NonProductionLicenseBanner.vue';
 import TrialOverBanner from '@/components/banners/TrialOverBanner.vue';
 import TrialBanner from '@/components/banners/TrialBanner.vue';
 import V1Banner from '@/components/banners/V1Banner.vue';
+import EmailConfirmationBanner from '@/components/banners/EmailConfirmationBanner.vue';
+import type { Component } from 'vue';
+import type { N8nBanners } from '@/Interface';
+
+// All banners that can be shown in the app should be registered here.
+// This component renders the banner with the highest priority from the banner stack, located in the UI store.
+// When registering a new banner, please consult this document to determine it's priority:
+// https://www.notion.so/n8n/Banner-stack-60948c4167c743718fde80d6745258d5
+export const N8N_BANNERS: N8nBanners = {
```

Comment:
The idea here is that banners are pushed to stack at different time points (app mounted, settings fetched, user cloud data fetched, ...) which do not have a preset order necessarily so we cannot rely on the order in which we call `pushBannerToStack`.
On the similar side, initial implementation was to always sort the stack based on priority and then this component always renders first banner from the stack, but our e2e tests were failing when this object is imported in store (as part of the `pushBannerToStack` action) file (because of the component imports), so I left this logic that determines highest-priority banner in this component here.